edx_name_affirmation/handlers.py

```diff
@@ -42,16 +42,19 @@ def idv_attempt_handler(attempt_id, user_id, status, photo_id_name, full_name, *
         photo_id_name(str): name to be used as verified name
         full_name(str): user's pending name change or current profile name
     """
+    trigger_status = VerifiedNameStatus.trigger_state_change_from_idv(status)
 
-    log.info('VerifiedName: idv_attempt_handler triggering Celery task for user %(user_id)s '
-             'with photo_id_name %(photo_id_name)s and status %(status)s',
-             {
-                 'user_id': user_id,
-                 'photo_id_name': photo_id_name,
-                 'status': status
-             }
-             )
-    idv_update_verified_name.delay(attempt_id, user_id, status, photo_id_name, full_name)
+    # only trigger celery task if status is relevant to name affirmation
+    if trigger_status:
```

Comment:
Can we also log the case where the trigger_status is not defined? Just do a log.info to explain that no celery task will be created because the IDV status is %status.